webv2/web.go

```diff
@@ -830,15 +830,15 @@ func getSourceAndTargetProfiles(sessionState *session.SessionState, details type
 			return profiles.SourceProfile{}, profiles.TargetProfile{}, utils.IOStreams{}, "", fmt.Errorf("error while creating config to initiate sharded migration:%v", err)
 		}
 	} else if sessionState.Driver == constants.CASSANDRA {
-		sourceProfileString = fmt.Sprintf("host=%v,port=%v,user=%v,password=%v,keyspace=%v,datacenter=%v",
+		sourceProfileString = fmt.Sprintf("\"host=%v\",\"port=%v\",\"user=%v\",\"password=%v\",\"keyspace=%v\",\"datacenter=%v\"",
 			sourceDBConnectionDetails.Host,
 			sourceDBConnectionDetails.Port,
 			sourceDBConnectionDetails.User,
 			sourceDBConnectionDetails.Password,
 			sessionState.DbName,
 			sourceDBConnectionDetails.DataCenter)
 	} else {
-		sourceProfileString = fmt.Sprintf("host=%v,port=%v,user=%v,password=%v,dbName=%v",
+		sourceProfileString = fmt.Sprintf("\"host=%v\",\"port=%v\",\"user=%v\",\"password=%v\",\"dbName=%v\"",
```

Comment:
We are adding quotes here. But the parsing logic in common.go doesn't remove these quotes. Does this code go through the logic written in above in `profiles/common.go` ? Have you tested this?